pkg/virt-api/webhooks/amd64.go

```diff
@@ -0,0 +1,27 @@
+package webhooks
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sfield "k8s.io/apimachinery/pkg/util/validation/field"
+
+	v1 "kubevirt.io/api/core/v1"
+)
+
+// ValidateVirtualMachineInstanceAmd64Setting is a validation function for validating-webhook on Amd64
+func ValidateVirtualMachineInstanceAmd64Setting(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec) []metav1.StatusCause {
+	var statusCauses []metav1.StatusCause
+	validateWatchdogAmd64(field, spec, &statusCauses)
+	return statusCauses
+}
+
+func validateWatchdogAmd64(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec, statusCauses *[]metav1.StatusCause) {
+	if spec.Domain.Devices.Watchdog != nil {
+		if spec.Domain.Devices.Watchdog.WatchdogDevice.I6300ESB == nil {
```

Comment:
I'm unsure, is the specific watchdog device already added at this stage? Or could `spec.Domain.Devices.Watchdog` still be pointing to an empty struct e.g.:
```
watchdog: {}
```